app/views/pensions/ReliefAtSourcePensionsView.scala.html

```diff
@@ -32,7 +32,7 @@
 @(form: Form[Boolean], taxYear: Int)(implicit user: User[_], messages: Messages, appConfig: AppConfig)
 
 @titleForUse = @{messages("pensions.reliefAtSourcePensions.title")}
```

Comment:
we should probably rename this variable if we're going to use the same one for title and heading.